packages/nouns-webapp/src/pages/Vote/index.tsx

```diff
@@ -340,39 +395,110 @@ const VotePage = ({
             <Card className={classes.voteCountCard}>
```

Comment:
Same for the vote cards, can we abstract away into `VoteCard`s? Vote page is getting crazy, let's do what we can to keep it legible.